website2/docs/schedulers-k8s-persistent-volume-claims.md

```diff
@@ -24,7 +24,9 @@ sidebar_label: Kubernetes Persistent Volume Claims (CLI)
 
 <br/>
 
-It is possible to leverage Persistent Volumes with custom Pod Templates. The CLI commands allow you to configure a Persistent Volume Claim (dynamically or statically backed) when you submit your topology. They also permit you to configure a Persistent Volume without a custom Pod Template. The CLI commands override any configurations you may have present in the Pod Template, but Heron's configurations will take precedence over all others.
+It is possible to leverage Persistent Volumes with custom Pod Templates but the Volumes you add will be shared amongst all Pods in the topology. Please leverage Pod Templates if you require Volumes which are shared amongst the entire topology.
+
+The CLI commands allow you to configure a Persistent Volume Claim (dynamically or statically backed) which will be unique and isolated to each Pod and mounted in a single `Executor` when you submit your topology. They also permit you to configure a Persistent Volume without a custom Pod Template which will be specific to an individual Pod. The CLI commands override any configurations you may have present in the Pod Template, but Heron's configurations will take precedence over all others.
```

Comment:
It would be good to describe why someone would use a persistent volume claim with Heron.

Reply:
It is essentially persistent storage that can be used for process checkpointing (no more bookkeeper?), caching, intermediate results which have analysis value (ETL to data lake or warehouse), data enrichment (dictionaries, databases), etc.

Reply:
Updated.

Reply:
I didn't realize this could be used for checkpointing.. This is very cool.

Reply:
Lightbulb has come on!

Reply:
> I didn't realize this could be used for checkpointing.. This is very cool.

I honestly cannot claim this idea as my own, I think this came up in a Slack channel conversation. It might have been Windham Wong's idea?